ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/CanHardFork.hs

```diff
@@ -190,7 +195,16 @@ byronTransition ByronPartialLedgerConfig{..}
                           then error "byronTransition: impossible"
                           else History.countSlots s endorsedInSlot
 
-    -- 'tryBumpVersion' assumes head of the list is the newest, so we do too
+    -- check if the candidate update protocol induces a hard fork to Shelley
+    bumpsMajorProtocolVersion :: CC.Update.CandidateProtocolUpdate -> Bool
+    bumpsMajorProtocolVersion upd =
+        get upd == shelleyMajorVersion
+      where
+        get :: CC.Update.CandidateProtocolUpdate -> Word16
+        get = CC.Update.pvMajor . CC.Update.cpuProtocolVersion
```

Comment:
I would rename it to `upd(Proposed)MajorVersion` as `get` is rather vague.

Reply:
I figured it was OK, since the use and defn are will within 4 lines with no other definitions in the same scope.

But I only did this to avoid an overly long line. So I inlined it with a newline instead.